nixos/modules/services/networking/i2pd.nix

```diff
@@ -79,7 +105,7 @@ let
       # This option is defined here because exploratory tunnels (in main config) do not support it
       lengthVariance = mkOption {
         type = types.int;
-        description = lib.mdDoc "Random number of hops to add or subtract to an ${name} tunnel between -3 and 3";
+        description = lib.mdDoc "Random number of hops to add or subtract to an ${name} tunnel.";
```

Comment:
Don't remove clarifications from descriptions (unless of course they're invalid), also descriptions shouldn't end with . (since your updating the module you can clean up any descriptions ending with .

Reply:
I checked the source code but found no checks other than that the minimum tunnel length must be greater than 1.